gui/src/loader.rs

```diff
@@ -225,12 +229,13 @@ impl Loader {
 
         if let Some(bitcoind) = &self.internal_bitcoind {
             bitcoind.stop();
-        } else if self.gui_config.start_internal_bitcoind {
+        } else if self.waiting_daemon_bitcoind && self.gui_config.start_internal_bitcoind {
             if let Ok(config) = Config::from_file(self.gui_config.daemon_config_path.clone()) {
                 if let Some(bitcoind_config) = &config.bitcoind_config {
                     let mut stopped = false;
                     while !stopped {
-                        stopped = stop_bitcoind(&bitcoind_config);
+                        stopped = stop_bitcoind(bitcoind_config);
+                        std::thread::sleep(std::time::Duration::from_secs(1))
```

Comment:
I think 1 second is too long and would trigger Gnome's "application isn't responding". Can you go for half a sec?